### ISSUE TITLE
Tweet mashup

### ISSUE THREAD

**Issue by chigozienri** [Karabonne/bot.garden#4]
Could you add an option to train on a mashup of multiple twitter feeds? (a la the dril/trump mashup here: https://github.com/minimaxir/tweet-generator)

**Comment by Karabonne** [comment]
yep! i've got an idea on how to implement this with markovify, will hopefully make some progress on it this week (sorry for the late reply, i thought github would notify me of an issue posted :/ )
